addon/modifiers/enable-tooltip.ts

```diff
@@ -0,0 +1,205 @@
+    iconI.className = icon;
+    titleContainer.append(iconI);
+  }
+  let titleSpan = document.createElement('span');
+  titleSpan.className = 'title';
+  _setElementContent(titleSpan, title, html);
+
+  titleContainer.append(titleSpan);
+
+  state.tooltip.append(titleContainer);
+
+  if (subtitle) {
+    let subtitleSpan = document.createElement('span');
+    subtitleSpan.className = 'subtitle';
+    _setElementContent(subtitleSpan, subtitle, html);
+    state.tooltip.append(subtitleSpan);
+  }
+  if (isTesting()) {
+    document.querySelector('#ember-testing')?.append(state.tooltip);
+  } else {
+    document.body.append(state.tooltip);
+  }
+}
+
+function _delayedCreate(state: EnableTooltipState, args: EnableTooltipArgs): void {
+  if (state.isRendered) return;
+  state.setTimeoutId = setTimeout(() => {
+    _create(state, args);
+  }, 300);
+}
+
+function _computePosition(state: EnableTooltipState, args: EnableTooltipArgs) {
+  const { placement } = args.named;
+  new Djoo().computePosition(
+    state.tooltip,
+    state.element,
+    {
+      placement: placement,
+      cssVariableName: 'modifier-tooltip',
+      elementTargetMargin: 9,
+      viewPortPadding: 16
+    },
+    { defaultRotation: 0, height: 4, width: 8 }
+  );
+}
+
+function _create(state: EnableTooltipState, args: EnableTooltipArgs): void {
+  if (state.isRendered) return;
+
+  _generateHTMLStructure(state, args);
+  _computePosition(state, args);
+
+  state.animation = createAnimation(state.tooltip, [{ opacity: 0 }, { opacity: 1 }], {
+    duration: 250,
+    fill: 'forwards'
+  });
+  state.animation.play();
+  state.isRendered = true;
+  state.setTimeoutId = null;
+}
+
+function _destroy(event: Event, state: EnableTooltipState): void {
+  if (state.setTimeoutId) {
+    clearTimeout(state.setTimeoutId);
+    state.setTimeoutId = null;
+    return;
+  }
+
+  const relatedTarget = (<MouseEvent>event).relatedTarget;
+  if (!state.isRendered || (relatedTarget instanceof Node && state.element.contains(relatedTarget))) return;
+
+  state.animation.reverse();
+  state.animation.finished.then(() => {
+    run(() => {
+      state.tooltip.remove();
+      state.isRendered = false;
+    });
+  });
+}
+
+function _setDefaultValue(args: EnableTooltipArgs): void {
+  args.named = {
+    ...args.named,
+    ...{
+      placement: args.named.placement || 'bottom',
+      trigger: args.named.trigger || 'hover focus',
+      title: args.named.title || '',
+      html: args.named.html || false
+    }
+  };
+}
+
+function _initEventListener(state: EnableTooltipState, element: HTMLElement, args: EnableTooltipArgs): void {
+  const { trigger } = args.named;
+  const splitTrigger = trigger.split(' ');
+
+  if (splitTrigger.includes('hover')) {
+    element.addEventListener('mouseover', () => {
+      _delayedCreate(state, args);
+    });
+
+    element.addEventListener('mouseout', (event) => {
+      _destroy(event, state);
+    });
+  }
+
+  if (splitTrigger.includes('focus')) {
```

Comment:
Oh ! are we displaying the tooltip on keyboard focus ???
Nice!

Reply:
This is the current case with JQuery tooltip too

Reply:
ah okay ! but jQuery sucks ! :trollface: